spec/factories/other_advice_method.rb

```diff
@@ -1,5 +1,6 @@
 FactoryGirl.define do
   factory :other_advice_method do
     name { Faker::Lorem.sentence }
+    sequence(:cy_name) { |n| "Dull gyngor arall #{n}" }
```

Comment:
:smile: My point is, if we don't care about `name` being unique, we shouldn't be caring about `cy_name`'s uniqueness or lack thereof. The reason I'm being pushy with this is that this kind of thing tends to get cargo-culted around the rest of the project, and we'll end up with sequences where they're not being used for any specific reason.